sdks/python/scripts/run_pylint.sh

```diff
@@ -104,6 +104,7 @@ ISORT_EXCLUDED=(
   "model.py"
   "taxi.py"
   "process_tfma.py"
+  "pubsub_test.py"
```

Comment:
What is the isort problem related to this file?

Reply:
It was insisting I place the protobuf import prior to the beam imports, but then if I did that, complaining about the grouping with the pubsub import.  However, as mentioned above, I didn't realise you could nest imports in a try/catch, so this should resolve this.

Reply:
Can this exclusion be removed now?

Reply:
Every time I try to remove it, unfortunately I get a pylint error.

Please see https://github.com/apache/beam/pull/9232#issuecomment-519044616 for details.